nomad/structs/errors.go

```diff
@@ -25,6 +26,8 @@ const (
 	ErrUnknownJobPrefix        = "Unknown job"
 	ErrUnknownEvaluationPrefix = "Unknown evaluation"
 	ErrUnknownDeploymentPrefix = "Unknown deployment"
+
+	errRPCCodedErrorPrefix = "RPC_ERROR::"
```

Comment:
A quick `rg --fixed-strings 'RPC Error: '` yielded no results, so we could use that instead. My concern is that I'd love to start using this pattern more widely, but it could easily leak into CLI or client agent logs if we aren't careful about our checks/conversions.
```suggestion
	errRPCCodedErrorPrefix = "RPC Error: "
```

Not a big deal either way. Users aren't going to get upset from seeing `RPC_ERROR::` once or twice if it ever even happens. :sweat_smile: